src/test/eo/org/eolang/collections/list-tests.eo

```diff
@@ -471,3 +471,53 @@
   assert-that > @
     ((res.at 0).length)
     $.equal-to 0
+
+[] > list-sorted
+  sorted. > res!
+    list
+      * 4 6 3 5
+  assert-that > @
+    res
+    $.equal-to
+      list
+        * 3 4 5 6
+
+[] > list-sorted-2
+  sorted. > res!
+    list
+      * 5 3 6 4
+  assert-that > @
+    res
+    $.equal-to
+      list
+        * 3 4 5 6
+
+[] > reversed-list-sorted
+  sorted. > res!
+    list
+      * 4 3 2 1
+  assert-that > @
+    res
+    $.equal-to
+      list
+        * 1 2 3 4
+
+[] > similar-elements-list-sorted
+  sorted. > res!
+    list
+      * 1 1 1
+  assert-that > @
+    res
+    $.equal-to
+      list
+        * 1 1 1
+
+[] > one-element-list-sorted
+  sorted. > res!
+    list
+      * 42
+  assert-that > @
+    res
+    $.equal-to
+      list
+        * 42
```

Comment:
@EugeneDar please, add corner test cases with an empty array and with negative numbers

Reply:
@Graur sure, I'll do it